buildkite/bazelci.py

```diff
@@ -468,6 +473,15 @@ def execute_commands(
     if use_bazel_at_commit and use_but:
         raise BuildkiteException("use_bazel_at_commit cannot be set when use_but is true")
 
+    bazel_version = config.get("bazel") or bazel_version
+    if bazel_version:
+        if use_bazel_at_commit or use_but:
```

Comment:
The idea is correct, but I think throwing an exception is too strict - if we run a project as a downstream project, it's possible that it has a bazel version specified in the config, but we should just ignore it in that case. Can you just move the os.environ[] line down to the else block in the try?